tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -0,0 +1,306 @@
+# from .common import *  # NOQA
```

Comment:
Remove this line. 

Clean up imports - makes them easier to read. 

Regular imports first `import pytest`, then `from .common`, etc.. (alphabetized)